modules/menu.js

```diff
@@ -0,0 +1,21 @@
+// decare variable to store menu links
+const menulinks = document.querySelectorAll('.row-list li a');
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] `required`
- [ ] amazing job so far 👏 
- [ ] Please enable the disabled linter as well 🥂

Reply:
Thank you, @mwanawabangona , for taking the time to review my pull request.